### ISSUE TITLE
request: handle resize escape sequence: `ESC [ 8;<height>;<width>t`

### ISSUE THREAD

**Issue by msg** [endaaman/tym#118]
Currently `tym` doesn't support handling the resize escape sequence `ESC [ 8;<height>;<width>t`.  It would be nice if was added.

**Comment by endaaman** [comment]
Hi @msg,

Thanks for requesting a new feature.
Are you refering to https://gnome.pages.gitlab.gnome.org/vte/gtk3/signal.Terminal.resize-window.html ?
This seems easy to introduce into tym. I'll implement it when I have some free time soon.

**Comment by msg** [comment]
I think so.  I am unfamiliar with `vte` from an API stand point.  I look forward to testing/using it.

**Comment by msg** [comment]
I tried your latest commit.  It seems to only handle the height change.  I tested by doing a:
```
$ printf "\e[8;40;161t"
```
And the height changes, but the width doesn't.

**Comment by endaaman** [comment]
Your command:

```
$ printf "\e[8;40;161t"
```

and this 

```
$ printf "\x1b[8;40;161t"
```

are both working on my env.

**Comment by msg** [comment]
Yeah, I tried it too.  It worked.  I probably fat fingered it.  This works great.  Thanks for adding it.

**Comment by endaaman** [comment]
Thanks for checking again. I'm going to release the new version soon.
